notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -4,7 +4,7 @@
       "cell_type": "code",
```

Comment:
<p>The steps performed include the following:</p><ul><li>Create a docker repository and config.</li><li>Create a custom container image with your customized training code.</li><li>Setup a service account and a Google Cloud Storage bucket.</li><li>Create and launch your custom training job with your custom container.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>